### DISCUSSION TITLE
Clip Path:  Cannot find name 'circle'

### DISCUSSION THREAD

**Discussion by asingleton6** [vanilla-extract-css/vanilla-extract#877] [Q&A] [answered]
I tried using the `clipPath: circle(50px at 0 100px)` but I get an error from TypeScript saying cannot find name circle. Has anybody else ran into this issue?

![image](https://user-images.githubusercontent.com/5387554/194949988-98301485-ce39-4be0-97d8-2c51fd551d9e.png)

**Comment by michaeltaranto** [comment, marked as answer]
Hey 👋 , when using built in CSS functions they need to be passed as a string value for the property (i.e. wrapped in quotes), so in this case:
```diff
export const finishCheckbox = style({
-  clipPath: circle("50px at 0 100px"),
+  clipPath: 'circle(50px at 0 100px)',
});
```
Also worth noting the values passed to the shape should not be quoted as per the [clip-path documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path).

**Reply by asingleton6** [reply]
Thank you very much for clear that up! This worked perfectly. 😃
